.github/actions/setup-terraform/action.yaml

```diff
@@ -10,6 +10,18 @@ inputs:
   batch-container-image-name:
     description: The name of the container image to use for the Batch job.
     required: true
+  batch-job-definition-vcpu:
+    description: >
+      Count of cVPUs to provision for the container. Per AWS requirements,
+      must be formatted as a float, e.g. 1.0 for 1 vCPU. The minimum
+      is 1 vCPU and values must be specified in increments of 0.25.
+    required: true
+  batch-job-definition-memory:
+    description: Count of megabytes of RAM to provision for the container.
+    required: true
```

Comment:
Adding CPU and memory requirements as inputs allows calling workflows more fine-grained control over their compute environment. While the condo model will likely use the same configuration for now, we could imagine exposing similar variables to allow the caller to specify a Fargate or EC2 backend, or CPU vs. GPU instances.

Reply:
Nice! Love the flexibility.